modules/transposition.nix

```diff
@@ -67,7 +67,11 @@ in
     perInput =
       system: flake:
       mapAttrs
-        (attrName: attrConfig: flake.${attrName}.${system})
+        (attrName: attrConfig:
+          flake.${attrName}.${system} or (throw ''
+            Attemt to access non existent attribute ${attrName}.${system} of flake ${flake}.
```

Comment:
"Attempt"

Reply:
Thanks!
I've mostly redone the error message in #206 to add more info and fix the problem with `${flake}`.